src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -6270,69 +6270,49 @@ private static string GetNamespaceToRemove(CompletionContext context, TypeComple
         internal static List<CompletionResult> CompleteHelpTopics(CompletionContext context)
         {
             var results = new List<CompletionResult>();
-            var searchPaths = new List<string>();
-            var currentCulture = CultureInfo.CurrentCulture.Name;
-
-            // Add the user scope path first, since it is searched in order.
-            var userHelpRoot = Path.Combine(HelpUtils.GetUserHomeHelpSearchPath(), currentCulture);
-
-            if (Directory.Exists(userHelpRoot))
-            {
-                searchPaths.Add(userHelpRoot);
-            }
-
-            var dirPath = Path.Combine(Utils.GetApplicationBase(Utils.DefaultPowerShellShellID), currentCulture);
-            searchPaths.Add(dirPath);
-
-            var wordToComplete = context.WordToComplete + "*";
-            var topicPattern = WildcardPattern.Get("about_*.help.txt", WildcardOptions.IgnoreCase);
-            List<string> files = new List<string>();
+            string userHelpDir = HelpUtils.GetUserHomeHelpSearchPath();
+            string appHelpDir = Utils.GetApplicationBase(Utils.DefaultPowerShellShellID);
+            string currentCulture = CultureInfo.CurrentCulture.Name;
+
+            //search for help files for the current culture + en-US as fallback
+            var searchPaths = new string[4]
+            { 
+                Path.Combine(userHelpDir, currentCulture),
+                Path.Combine(userHelpDir, "en-US"),
+                Path.Combine(appHelpDir, currentCulture),
+                Path.Combine(appHelpDir, "en-US")
```

Comment:
For discussion:
```suggestion
                Path.Combine(userHelpDir, currentCulture),
                Path.Combine(appHelpDir, currentCulture),
                Path.Combine(userHelpDir, "en-US"),
                Path.Combine(appHelpDir, "en-US")
```
Maybe users want to get topics in their language first?